### ISSUE TITLE
"Mobile Techniques proposed for WCAG 1.4.3" perhaps redundant?

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/Mobile-A11y-Extension#4]
In current https://w3c.github.io/Mobile-A11y-Extension/ there is a proposed technique

> [MOBILE] Color Contrast: The default point size for mobile platforms might be larger than the default point size used on non-mobile devices. When determining which contrast ratio to follow, developers should strive to make sure to apply the lessened contrast ratio only when text is roughly equivalent to 1.2 times bold or 1.5 times (120% bold or 150%) that of the default platform size. 

Unless I'm missing something, this technique doesn't actually say anything different from the regular color contrast advice, with exception of perhaps talking about the "default point size"  (which is already a bit contentious due to its use of `points` rather than `px`, and is - from what I understand - slated to be reviewed for future WCAG iterations). Therefore, I'd suggest dropping this, as it adds nothing in my view.

**Comment by kwahlbin** [comment]
The low vision taskforce will be taking the lead on this
